services/explorer/src/components/Footer/Footer.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { Footer as TFooter } from '@taraxa_project/taraxa-ui';
 import { useFooterEffects } from './Footer.effects';
 
-export const Footer = () => {
+export const Footer = (): JSX.Element => {
```

Comment:
Why do we need the type here? And if we add the type, I think it should be FC imported from React, not JSX.Element.

Reply:
FC does need much more than what we have in most of these propless components: `Type 'Element' provides no match for the signature '(props: {}, context?: any): ReactElement<any, any>'.`

Reply:
Got it. Ok then but if we have props then we need to switch to FC<PropInterface>